rel/files/riak-cs-debug

```diff
@@ -72,24 +88,42 @@ dump () {
 
 usage () {
 cat << 'EOF'
-riak-cs-debug: Gather info from a node for troubleshooting. See 'man riak-cs-debug'.
-
-Usage: riak-cs-debug [-c] [-l] [-r] [-s] [-e] [FILENAME | -]
-
--c, --cfgs           Gather Riak CS configuration information.
+Usage: riak-cs-debug [-c] [-l] [-r] [-s] [-e] [-v] [FILENAME | -]
+
+-c, --cfgs           Gather Riak configuration information (includes everything
+                     in platform_etc_dir).
+                     Please see the Privacy Note below.
+    --ssl-certs      Do not skip the capture of *potentially private* SSL
+                     certificates.
+                     By default files in the platform_etc_dir with the following
+                     extensions are not included in the riak-cs-debug archive:
+                     .pfx, .p12, .csr, .sst, .sto, .stl, .pem, .key.
+                     Please see the Privacy Note below.
 -l, --logs           Gather Riak CS logs.
 -r, --riakcmds       Gather Riak CS information and command output.
 -s, --syscmds        Gather general system commands.
--e, --extracmds      Gather extra command output. These commands are too intense to
-                     run on all nodes in a cluster but appropriate for a single node.
-    --log-mtime      Determine last modified time as n*24 hours, which riak-cs-debug gathers
-                     newer logs than the time. (default: 14)
+-e, --extracmds      Gather extra command output. These commands are too intense
+                     to run on all nodes in a cluster but appropriate for a
+                     single node.
+-v, --verbose        Print verbose failure messages to stdout
```

Comment:
Useful option :beetle: 
The code below seems that messages are output to std*err*, right?